backend/spec/services/spreadsheet_generator_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "roo"
+require "tempfile"
+
+RSpec.describe SpreadsheetGenerator do
+  describe "#perform" do
+    let(:enrollment) { create(:enrollment, :franceconnect) }
+    let(:foreign_enrollment) { create(:enrollment, :api_entreprise) }
```

Comment:
```suggestion
    let(:enrollment_1) { create(:enrollment, :franceconnect) }
    let(:enrollment_2) { create(:enrollment, :api_entreprise) }
```

foreign n'a pas de sens dans ce contexte.